command/agent.go

```diff
@@ -124,15 +125,22 @@ func (c *AgentCommand) Flags() *FlagSets {
 	})
 
 	f.StringVar(&StringVar{
-		Name:       "log-level",
+		Name:       flagNameLogLevel,
 		Target:     &c.flagLogLevel,
 		Default:    "info",
-		EnvVar:     "VAULT_LOG_LEVEL",
+		EnvVar:     EnvVaultLogLevel,
 		Completion: complete.PredictSet("trace", "debug", "info", "warn", "error"),
 		Usage: "Log verbosity level. Supported values (in order of detail) are " +
 			"\"trace\", \"debug\", \"info\", \"warn\", and \"error\".",
 	})
 
+	f.StringVar(&StringVar{
+		Name:   flagNameLogFile,
+		Target: &c.flagLogFile,
+		EnvVar: EnvVaultLogFile,
+		Usage:  "Path to the log file that Vault should use for logging",
```

Comment:
It's more than this though right?  It can be a log dir, or a prefix: https://developer.hashicorp.com/consul/docs/agent/config/cli-flags#_log_file

Reply:
Shouldn't we add the other log options (like the rotation options) as well as flags?  Or alternatively, since unlike consul we have logs of config without associated flags, should we maybe not add the -log-file CLI option?

Reply:
In Consul yes, as Consul does rotation of logs etc. so it fiddles with the name a lot more. In Vault I took the simple approach so it's no frills.